src/communication/serverMessages.ts

```diff
@@ -1,4 +1,5 @@
 import { FSharpFormat } from "../configuration/FSharpConfiguration";
+let log = require("loglevel");
```

Comment:
In type script you should use the following syntax: import log = require("loglevel");
[https://www.typescriptlang.org/docs/handbook/migrating-from-javascript.html](doc link)